native/cocos/renderer/gfx-wgpu/WGPUCommandBuffer.cpp

```diff
@@ -446,7 +437,7 @@ void CCWGPUCommandBuffer::bindStates() {
         if (indexBuffer) {
             wgpuRenderPassEncoderSetIndexBuffer(_gpuCommandBufferObj->wgpuRenderPassEncoder,
                                                 indexBuffer->gpuBufferObject()->wgpuBuffer,
-                                                indexBuffer->getStride() == 2 ? WGPUIndexFormat::WGPUIndexFormat_Uint16 : WGPUIndexFormat_Uint32,
+                                                indexBuffer->getStride() <= 2 ? WGPUIndexFormat::WGPUIndexFormat_Uint16 : WGPUIndexFormat_Uint32,
```

Comment:
We might `assert(indexBuffer->getStride() >= 2)`. I am not sure whether we can handle `stride < 2`.

Reply:
OK, I think I must have met uint8 array as an index buffer, shall we make all index buffer a u16array or a u32array?